src/main/java/pl/tomaszdziurko/jvm_bloggers/JvmBloggersConfiguration.java

```diff
@@ -33,4 +34,8 @@ public Client getMailingRestClient(@Value("${mailing.apiKey}") String malingApiK
         return client;
     }
 
+    @Bean
+    public CacheManager cacheManager() {
+        return new GuavaCacheManager();
```

Comment:
@goostleek Chill :) What I meant by KISS in the example of yours is to use "The JDK-based Cache implementation resides under org.springframework.cache.concurrent package. It allows one to use ConcurrentHashMap as a backing Cache store." Please see: 35.6.1 JDK ConcurrentMap-based Cache

- You do not have to.
- Yes, this is what we are doing right now

I think there is a bit of misunderstanding here :)